ha/k8s_setup/ha_start.py

```diff
@@ -13,7 +15,8 @@
                                     )
 
 my_parser.add_argument('--services', '-s', help='service_name', required=True, action='store')
-my_parser.add_argument('--config', '-c', help='config file', required=False, action='store')
+my_parser.add_argument('--config', '-c', help='config file url', required=False, action='store')
+my_parser.add_argument('--debug', '-d', help='debug level {0|1}', type=int, nargs='?', const=1, default=0, required=False, action='store')
```

Comment:
Can this argument also be added conditionally ? i.e only when buld is debug build

Reply:
done